bccsp/sw/keyimport.go

```diff
@@ -33,7 +33,9 @@ func (*aes256ImportKeyOptsKeyImporter) KeyImport(raw interface{}, opts bccsp.Key
 		return nil, fmt.Errorf("Invalid Key Length [%d]. Must be 32 bytes", len(aesRaw))
 	}
 
-	return &aesPrivateKey{utils.Clone(aesRaw), false}, nil
```

Comment:
Just curious. Why do we need to clone the byte slice? Do we expect the `raw` to get changed?

Reply:
I don't think we need to "clone" here either